docs/layers.md

```diff
@@ -347,9 +490,261 @@ Implied but not stated: `source`: `openstreetmap.org`.
+#### POI kind values:
+
+* `accountant`
+* `adit`
+* `administrative`
+* `advertising_agency`
+* `aerodrome`
+* `airport`
+* `alcohol`
+* `alpine_hut`
+* `ambulatory_care`
+* `amusement_ride`
+* `animal`
+* `aquarium`
+* `archaeological_site`
+* `architect`
+* `are_home`
+* `artwork`
+* `assisted_living`
+* `association`
+* `atm`
+* `attraction`
+* `aviary`
+* `bakery`
+* `bank`
+* `bar`
+* `beach`
+* `beacon`
+* `bed_and_breakfast`
+* `bench`
+* `bicycle_parking`
+* `bicycle_rental`
+* `bicycle`
+* `biergarten`
+* `block`
+* `boat_rental`
+* `bollard`
+* `books`
+* `brewery`
+* `bus_station`
+* `bus_stop`
+* `butcher`
+* `cafe`
+* `camp_site`
+* `car_repair`
+* `car_sharing`
+* `car`
+* `caravan_site`
+* `carousel`
+* `carpenter`
+* `cave_entrance`
+* `chalet`
+* `childcare`
+* `childrens_centre`
+* `cinema`
+* `clinic`
+* `closed`
+* `clothes`
+* `club`
+* `college`
+* `community_centre`
+* `company`
+* `computer`
+* `confectionery`
+* `consulting`
+* `convenience`
+* `courthouse`
+* `cross`
+* `dairy_kitchen`
+* `day_care`
+* `dentist`
+* `department_store`
+* `doctors`
+* `doityourself`
+* `dressmaker`
+* `drinking_water`
+* `dry_cleaning`
+* `dune`
+* `educational_institution`
+* `egress`
+* `electrician`
+* `electronics`
+* `embassy`
+* `emergency_phone`
+* `employment_agency`
+* `enclosure`
+* `estate_agent`
+* `fashion`
+* `fast_food`
+* `ferry_terminal`
+* `financial`
+* `fire_station`
+* `fishing`
+* `fitness_station`
+* `fitness`
+* `florist`
+* `food_bank`
+* `ford`
+* `foundation`
+* `fuel`
+* `gardener`
+* `gas`
```

Comment:
Is it worth adding some documentation here to make it clear that this is for bottled (camping) gas, rather than liquid (car) gas? And that the latter is called `fuel` here?